lib/nerves_hub_link/client/default.ex

```diff
@@ -77,4 +77,29 @@ defmodule NervesHubLink.Client.Default do
   def identify() do
     Logger.info("[NervesHubLink] identifying")
   end
+
+  @impl NervesHubLink.Client
+  def request_location() do
+    case Whenwhere.asks() do
+      {:ok, resp} ->
+        payload = %{
+          source: "whenwhere",
+          latitude: Float.parse(resp.body["latitude"]),
+          longitude: Float.parse(resp.body["longitude"])
```

Comment:
It seems like the `Float.parse/1` calls should be in the `Whenwhere` library. `Float.parse/1` returns a tuple.

Reply:
urg, can't believe i missed that